src/Picasso_UniformMesh.hpp

```diff
@@ -214,13 +256,15 @@ struct is_uniform_mesh
 
 //---------------------------------------------------------------------------//
 // Creation function.
-template <class MemorySpace>
+template <class MemorySpace, class ExecSpace>
```

Comment:
This feels odd, but I don't have a better way since nodes are built in the constructor